pkg/kubelet/cm/topologymanager/policy_options_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	pkgfeatures "k8s.io/kubernetes/pkg/features"
 )
 
-var fancyBetaOption = "fancy-new-option"
+var fancyAlphaOption = "fancy-new-option"
```

Comment:
why demote from beta to alpha?

Reply:
Cause we don't have any `alpha` features when we are graduating `prefer-closest-numa` to beta and I need at least one `alpha` feature to test out corner cases.

Reply:
ok, can we add (vs replace) a `fancyBetaOption` to be able to increase the test coverage?

Reply:
I have changed the code according to your review.